backend/compilers/gcc2.6.3-mipsel/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "platform": "ps1",
+    "cc": "\"${COMPILER_DIR}\"/gcc -G0 -c -B \"${COMPILER_DIR}\"/ $COMPILER_FLAGS \"$INPUT\" -o \"$OUTPUT\""
```

Comment:
`-G0` is exposed as an option, so hard-coding here means users can't try out e.g. -G4/_G8